ui_framework/components/menu/_menu.scss

```diff
@@ -1,12 +1,26 @@
+/**
+ * padding-left: 0 and list-style: none rules
+ * are meant for ul.kuiMenu and li.kuiMenuItem
+ * elements.
+ *
+ * kuiMenu and kuiMenuItems can classify both
+ * nonsemantic divs as well as lists (ul, li).
+ */
+.kuiMenu {
+  padding-left: 0;
+}
+
 .kuiMenu--contained {
   border: $globalBorderThin;
+  padding-left: 0;
 
   .kuiMenuItem {
     padding: 6px 10px;
   }
 }
 
 .kuiMenuItem {
+  list-style: none;
```

Comment:
And this would become:

```scss
/**
 * 1. Allow this class to be applied to `li` elements.
 */
.kuiMenuItem {
  list-style: none; /* 1 */
}
```